src/styles/components/_breadcrumbs.scss

```diff
@@ -12,19 +12,12 @@
 		display: inline-flex;
 		align-items: center;
 		margin-bottom: $spacer-xxs;
-
-		p,
-		a {
-			overflow: hidden;
-			text-overflow: ellipsis;
-			display: flex;
-			-webkit-box-orient: vertical;
-		}
 	}
 
 	&__link {
 		font-size: $font-size-xs;
-		display: inline-flex;
+		/* stylelint-disable-next-line value-no-vendor-prefix */
+		display: -webkit-box;
```

Comment:
Zijn we zeker dat `-webkit-box` op alle browsers werkt? Is er geen alternatief?

Reply:
In de utilities (_text.scss) die reeds in het project zat, wordt dat ook op die manier gedaan. Ik heb tot nu toe nog geen werkend alternatief gevonden om dit te fixen. 
https://caniuse.com/?search=-webkit-box